### ISSUE TITLE
"Print Text" page will not display notes

### ISSUE THREAD

**Issue by alanlenzi** [oracc/oracc#111]
The "print text" page still does not display the in-line notes from the ATF files (#note:). The note markers appear. But on mouse-over, there is a java script problem. The regular corpus page displays the notes fine. 

After comparing the html code for one text (e.g., P497370) in both the [pager format](http://oracc.museum.upenn.edu/caspo/akkpm/P497370) and ["print text" format](http://oracc.museum.upenn.edu/caspo/akkpm/P497370/html), I noticed that there is one major difference in the code of the header section of the two pages. The "print text" page's header section does not have this line of code:

`<script src="https://code.jquery.com/jquery-3.2.1.min.js" type="text/javascript"><![CDATA[ ]]></script>
`

Perhaps I'm wrong. But I think all the "print text" pages will display the notes in the ATF files if this line is inserted into the header section of the template for the "print text" page. This seems like an easy fix.

This is an important issue since those of us who embed the "print text" page in their own website really need the notes to work. Thanks for your attention to this matter.

**Comment by alanlenzi** [comment]
This issue still affects the functionality and presentation of my two projects. Is there a timeline for a fix?

**Comment by alanlenzi** [comment]
I am submitting a grant proposal and really need the functionality that is not working when I embed output from ORACC on my own web site. Without the notes being visible, grant evaluators will not see much of the work that I am doing, which is not just transliteration and translation. Is there an ETA on fixing this functionality?

**Comment by jnovotny-lmu** [comment]
@stinney, 
You started working on a PDF print instead of the 'print' page functionality. Perhaps that would be a better fix since the output of a PDF from the 'print' page is not great, even though the notes print at the bottom of the page. Maybe that would be better for Alan, and  all users of Oracc.

**Comment by alanlenzi** [comment]
Is anyone monitoring this page? If ORACC is to be considered a viable resource then users should at least reasonably expect the courtesy of a response. I appreciate the tools ORACC makes available. But I thought there would be some support for those of us who have a long term commitment to contribute. It's been nine months without _any_ response.

**Comment by jnovotny-lmu** [comment]
@alanlenzi 
Yes, you will see that I responded on 5 Feb to your 5 Feb post, as you can see from the feed above. 
@stinney 
Would it be possible to address this issue ASAP or implement quickly? Alternatively, the print PDF feature that you created while in Munich the last time might be the best fix, if that can be completed quickly.

**Comment by alanlenzi** [comment]
@jnovotny-lmu
I saw your response before. I just didn't realize that you were part of the official ORACC team. I knew you were a major contributor, of course. I apologize for being short. I've been dealing with this notes issue since I started my own web site for my project (2015?) and began using the "embed" feature from ORACC. I have tried to work around it by loading the ATF files from ORACC with PHP and then use a java script and a modified CSS to convert the ATF locally. (Enrique Jiménez shared a "how-to" about this a few years back.) But, I have not had great success implementing this fix because my coding skills are pretty rudimentary. The notes still don't function properly, and the ORACC CSS always messes up my local CSS, despite my best attempts. That's why I tried to track down the problem in the "print text" page template (see original post). That one missing line of code seems to be the significant difference between the pager output and the print page output.
